src/main/java/it/gov/pagopa/debtposition/service/payments/PaymentsService.java

```diff
@@ -53,13 +53,13 @@ public class PaymentsService {
     @Value("${nav.aux.digit}")
     private String auxDigit;
 
-    public PaymentOption getPaymentOptionByIUV(@NotBlank String organizationFiscalCode,
-                                               @NotBlank String iuv) {
+    public PaymentOption getPaymentOptionByNAV(@NotBlank String organizationFiscalCode,
+                                               @NotBlank String nav) {
```

Comment:
add useful comment "passed IUV to NAV for temporary backward compatibility"

Reply:
done